spec/mailers/candidate_mailer_ucas_match_initial_email_spec.rb

```diff
@@ -33,7 +33,7 @@
 
     it_behaves_like(
       'a mail with subject and content',
-      I18n.t!('candidate_mailer.ucas_match.multiple_acceptances.subject'),
+      I18n.t!('candidate_mailer.ucas_match.multiple_acceptances.subject', withdraw_by_date: '1 February 2021'),
```

Comment:
I wonder if we should refactor email specs and not use the locale to test them, as we are hiding what the actual subject is meant to be. Maybe on another PR though.

Reply:
Yes, good point. I will add a commit in the next PR